public/characterCreator.html

```diff
@@ -1 +1,107 @@
+        <label>Physical Prowess</label>
+        <input type="number" id="pProwess" name="pProwess" value="0" class="stats"><br><br>
+        <label>Physical Endurance</label>
+        <input type="number" id="pEndurance" name="pEndurance" value="0" class="stats"><br><br>
+        <label>Speed</label>
+        <input type="number" id="speed" name="speed" value="0" class="stats"><br><br>
+        <label>Wealth</label>
+        <input type="number" id="wealth" name="wealth" value="0"><br><br>
+        <label>Power Name</label>
+        <input type="text" id="powername" name="powername"><br><br>
+        <label>Power Description</label>
+        <input type="text" id="powerdescription" name="powerdescription"><br><br>
+        <input type="submit" value="Create">
+    </form>
+    <button onclick="preview()">Preview</button>
+    <script type="text/javascript">
+    function preview(){
+      const elements = document.getElementsByClassName("stats");
+      document.getElementById('deficitpoints').innerHTML = calculateDeficit(elements)
+
+
+    }
+    /** Returns the cost of statpoints for the set level of the stat. Returns a negative number to indicate a gain of statpoints
+    * @param {number} setstat The level the stat is set to
+    * @return {number} result This is the resulting cost of statpoints for the set stat, negative if its a gain of statpoints
+    */
+    function generateStatCost(setStatLevel){
+      setStatLevel = parseInt(setStatLevel);
+      if(setStatLevel < 0){  //Checks if setstatlevel is negative if so returns the negative setstatlevel as a statpoint gain
+        return setStatLevel
+      } else{
+      for (loopIterator = 0, result = 0; loopIterator <= setStatLevel; loopIterator++){  // iterates to find the statpoint cost needed to get a stat to the setstat level
+        result = result +loopIterator;
+      }
+      return result; // returns statpoint cost in the case of it being a non negative.
+            }
+    }
+    /** Returns the net cost of set statlevels with current ammount of statpoitns, returns negative if their is a deficit and positive if there is a suprlus of statpoints
+    *@param {htmlcollection} statCollection This is a collection of elements that contain number values within the class 'stats'
+    *@return {number} workingValue This is a number that is the total statpoint cost being either negative or positive depending on if free statpoints remain.
+    */
+    function calculateDeficit(statCollection){
+      let workingValue = 0;
+      for(statValue of statCollection){
+        if(isNaN(statValue.value)){
+          return "Invalid entry";
+        }
+        workingValue += generateStatCost(statValue.value);
+      }
+      const setStatPoints = parseInt(document.getElementById('statpoints').value);
+      workingValue = setStatPoints-workingValue;
+      return workingValue;
+    }
+
+    </script>
+    <h3>Statpoints</h3>
```

Comment:
https://www.w3.org/MarkUp/html3/headings.html#:~:text=HTML%20defines%20six%20levels%20of,level%20and%20H6%20the%20least.

Reply:
I meant for this to be a link to a wcag doc, but it didn't link to that page, you should look up aria/ accessibility for semantic meaning for elements